docs/guides/primitives-rest-api.mdx

```diff
@@ -479,6 +479,141 @@ else:
   </TabItem>
 </Tabs>
 
+<span id="start-sampler-parms"></span>
+## Sampler primitive with REST API and parameterized circuits
+
+### 1. Initialize the account
+
+Because Qiskit Runtime Sampler is a managed service, you first need to initialize your account. You can then select the device you want to use to run your calculations on.
+
+Find details on how to initialize your account, view available backends, and invalidate tokens in this [topic](./setup-channel#set-up-to-use-ibm-quantum-platform-with-rest-api).
+
+### 2. Define parameters
+
+```python
+import requests
+import qiskit_ibm_runtime
+# Imports from Qiskit Runtime
```

Comment:
This is noisy, and it's not even true. The line above is from Runtime and the last line is not.
```suggestion
```